### DISCUSSION TITLE
Drawing track piece from vertical drop on twister

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16475] [Feature requests]
I know this would be up for debate in the community, but is it possible we could have two flat to steep on twister (unprecedented) at least with cheats?
here is a picture of what I mean

![what](https://user-images.githubusercontent.com/94884086/150292736-f2cd5737-5e02-49d2-bdd3-abc2aad76f6e.png)

**Comment by TheAtomTwister** [comment]
I agree.  that would be useful for sure when your building tight layouts.  I would really also like to see the Vertical Drop coaster also have the extended steep to flat as that will really help keep the G forces down on 200 foot vertical drops.

**Reply by 733737** [reply]
my main concern is whether both of these suggestions would be relegated to the dropdown special list. The multidimension coaster proves we can add elements to the construction window

**Reply by Broxzier** [reply]
Introducing a new button for the small and big pieces would be better than "special" dropdown list, imo.

**Reply by 733737** [reply]
i think it would be a great addition to 4.0
